packages/core/src/Table/TableRow.js

```diff
@@ -1,45 +1,75 @@
 import propTypes from '@dhis2/prop-types'
 import cx from 'classnames'
-import React, { useContext } from 'react'
-import css from 'styled-jsx/css'
-import { TableContext } from './TableContext'
-
-const tableRowStyles = css`
-    .zebraStriping:nth-child(even) {
-        background: #fbfcfd;
-    }
-`
+import React, { useState, forwardRef } from 'react'
+import { DragHandleCell } from './TableRow/DragHandleCell'
+import { ExpandedRowContent } from './TableRow/ExpandedRowContent'
+import { ExpandToggleCell } from './TableRow/ExpandToggleCell'
+import styles from './TableRow/TableRow.styles.js'
 
 /**
  * @module
  * @param {TableRow.PropTypes} props
  * @returns {React.Component}
- * @example import { TableRow } from '@dhis2/ui-core'
- * @see Live demo: {@link /demo/?path=/story/table--static-layout|Storybook}
+ * @example import { TableRow } from '@dhis2/ui'
+ * @see Live demo: {@link /demo/?path=/story/table--default|Storybook}
  */
-export const TableRow = ({
-    children,
-    className,
-    dataTest,
-    suppressZebraStriping,
-}) => {
-    const {
-        suppressZebraStriping: suppressZebraStripingFromContext,
-    } = useContext(TableContext)
-
-    const zebraStriping =
-        typeof suppressZebraStriping !== 'undefined'
-            ? !suppressZebraStriping
-            : !suppressZebraStripingFromContext
-
-    return (
-        <tr className={cx(className, { zebraStriping })} data-test={dataTest}>
-            {children}
-
-            <style jsx>{tableRowStyles}</style>
-        </tr>
-    )
-}
+export const TableRow = forwardRef(
+    (
+        {
+            children,
+            className,
+            dataTest,
+            expandableContent,
+            selected,
+            draggable,
+        },
+        ref
+    ) => {
+        const [isExpanded, setIsExpanded] = useState(false)
```

Comment:
This relates to the comment I placed here https://github.com/dhis2/ui/pull/463/files#r576220806. Here I've actually taken the opposite decision and I'm very unsure about this part.

The trade-off I considered was as follows:
- To control the expandable content behaviour via props we'd need:   
    1. The current `expandableContent` prop which determines what the content actually is
    2. A new `showExpandableContent` (boolean) prop to toggle visibility
    3. A new `onExpandHandleClick` (function) to handle the icon click (and probably toggle the `showExpandableContent` prop from the parent component)
- I then considered how much of a practical use case this would be, and couldn't really think of any.

So considering the above, I concluded a bit of internal state wasn't too bad since it would enable us to have a smaller component API.